src/Components/Entry/NewEntry.tsx

```diff
@@ -16,8 +19,9 @@ interface props {
 };
 
 const NewEntry = ({accountId}:props) => {
+    const dispatch = useDispatch();
     const [entryType, setEntryType] = useState<string>("User");
-    const [amount, setAmount] = useState<number|null>(null);
+    const [amount, setAmount] = useState<number|string>("");
```

Comment:
string?

Reply:
Null?

Reply:
Good point! We wouldn't want anyone adding a string to this form. I'll set to null.